### ISSUE TITLE
No sound on Ubuntu Bionic (ElementaryOS)

### ISSUE THREAD

**Issue by ungauss** [unitystation/unitystation#3007]
## Description
When running the Linux executable from https://unitystation.org/, I can hear no sound except the music that plays during the loading screen. All other aspects of the game work.

### Detailed Bug Report
[Player.log](https://github.com/unitystation/unitystation/files/4178776/Player.log)
Note: `ALSA snd_pcm_writei error: Broken pipe`

It's likely a driver issue, but I was told to open a GH issue anyways.

**Comment by Elijahrane** [comment]
Do you have pulseaudio installed?

**Comment by ungauss** [comment]
Yes. I also tried clearing it's config files and restarting, and installing a new version.

Is there a specific version I should try?

**Comment by PerfectTangent** [comment]
we've entirely changed how audio is managed in the game. Is this issue still relevant?
